packages/sanity/src/structure/panes/StructureToolPane.tsx

```diff
@@ -73,6 +79,7 @@ export const StructureToolPane = memo(
             pane={pane}
           />
         </Suspense>
+        <StructureTitle resolvedPanes={resolvedPanes} />
```

Comment:
Do we really need to move this here? It was moved from `StructureTool` to here, but it's using the same data. 
` const {resolvedPanes} = useResolvedPanes()`

Maybe this was kept from a previous change and it's not necessary anymore to move it.

Reply:
Yep, I think you're absolutely right about why this has moved 😅. I'll take a look at moving it back.